frontend/src/employee-frontend/components/unit/tab-unit-information/DaycareAclAdditionModal.tsx

```diff
@@ -64,12 +76,27 @@ export default React.memo(function DaycareAclAdditionModal({
   const { i18n } = useTranslation()
 
   const [formData, setFormData] = useState<DaycareAclAdditionFormState>({
+    type: 'PERMANENT',
+    firstName: '',
+    lastName: '',
     selectedEmployee: null,
     selectedGroups: null,
-    hasStaffOccupancyEffect: null
+    hasStaffOccupancyEffect: null,
+    pinCode: ''
   })
 
   const submit = useCallback(async () => {
+    if (formData.type === 'TEMPORARY') {
```

Comment:
First selecting a normal employee and then creating and sending temporary employee sends the normal employee in the POST payload

Reply:
Cannot reproduce :thinking: